packages/@webex/internal-plugin-metrics/src/new-metrics.ts

```diff
@@ -145,6 +145,36 @@ class Metrics extends WebexPlugin {
     throw new Error('Not implemented.');
   }
 
+  /**
+   * Returns a promise that will resolve to fetch options for submitting a metric.
+   * @public
+   * @param {Object} arg
+   * @param {String} arg.name - event name
+   * @param {Object} arg.payload - event payload
+   * @param {Object} arg.options - other options
+   * @returns {Promise} promise that resolves to options to be used with fetch
+   */
+  public async prepareMetricFetchOptions({
+    name,
+    payload,
+    options,
+  }: {
+    name: ClientEvent['name'];
+    payload?: RecursivePartial<ClientEvent['payload']>;
+    options: SubmitClientEventOptions;
+  }) {
+    // TODO: what to do with this?
+    // this.callDiagnosticLatencies.saveTimestamp({
+    //   key: name,
+    //   options: {meetingId: options?.meetingId},
+    // });
```

Comment:
We can't save a timestamp here, because it would be the time that the options were prepared, not when the fetch happens.  The fetch will happen in cantina, though, and I'm not sure I could/should call saveTimestamp from there.

Reply:
You can remove this, we won't use it in this case.

Reply:
Done